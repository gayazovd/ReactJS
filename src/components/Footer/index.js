import React, {PureComponent} from 'react';
import styles from './style.scss';
import classNames from 'classNames/bind';

const cx = classNames.bind(styles);

const Footer = () => 
    <div className={cx('wrapper')}>
       <div className={cx('logo')}>netflixroullete</div>
    </div>

export default Footer;