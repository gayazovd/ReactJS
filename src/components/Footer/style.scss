.logo{
  color: rgb(207, 63, 87);
  width: 70%;
  padding: 5px;
}
.wrapper{
    width: 100%;
    background: #616161;
    height: 40px;
    flex: 0 0 auto;
}