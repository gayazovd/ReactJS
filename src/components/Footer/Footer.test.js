import React from 'react';
import { shallow } from 'enzyme';
import Footer from './index';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Footer is render',() => {
    const footer = shallow(<Footer />);
    it('Footer snapshot', () => {
        expect(footer).toMatchSnapshot();
    })

    it('render Footer',() => {
        expect(footer.find('.logo').text()).toEqual('netflixroullete');
    })
})