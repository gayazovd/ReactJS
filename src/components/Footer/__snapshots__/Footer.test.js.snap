// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer is render Footer snapshot 1`] = `
<div
  className="wrapper"
>
  <div
    className="logo"
  >
    netflixroullete
  </div>
</div>
`;
